build/common/installer/scripts/tomlparser-agent-config.rb

```diff
@@ -351,6 +352,17 @@ def populateSettingValuesFromConfigMap(parsedConfig)
         puts "Using config map value: AZMON_MULTILINE_ENABLED = #{@multiline_enabled}"
       end
 
+      if !@controllerType.nil? && !@controllerType.empty? && @controllerType.strip.casecmp(@daemonset) == 0 && @containerType.nil?
+        resource_optimization_config = parsedConfig[:agent_settings][:resource_optimization]
```

Comment:
@pfrcks - our plan to rollout the resource optimizations  for linux through toggles, right? Please validate the setting the environment variable from yaml works as well, This way, toggle can set the environment variable to enable this feature.

Reply:
yes this will be done as part of rollout